notification/src/main/java/com/kyriba/conference/notification/domain/EmailNotification.java

```diff
@@ -0,0 +1,58 @@
+package com.kyriba.conference.notification.domain;
+
+import com.kyriba.conference.notification.domain.dto.EmailMessageOptions;
+import com.kyriba.conference.notification.domain.dto.EmailNotificationMessage;
+import com.kyriba.conference.notification.domain.dto.MessageType;
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Set;
+
+@Entity
+@Table(name = "email_notification",
+        indexes = @Index(name = "IX_EMAILNOTIFICATION_INFO", columnList = "notification_info_id"),
+        uniqueConstraints = @UniqueConstraint(name = "UK_EMAILNOTIFICATION_INFO", columnNames = {"notification_info_id"}))
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class EmailNotification implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @OneToOne(cascade = CascadeType.PERSIST, fetch = FetchType.EAGER)
+    @JoinColumn(name = "notification_info_id", referencedColumnName = "id", nullable = false)
+    private NotificationInfo notificationInfo;
+
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Don't specify fetch strategy in config, do it in runtime on request with query or graphs.